Dockerfile

```diff
@@ -1,27 +1,46 @@
-FROM golang:1.15.5 as builder
+# Image page: <https://hub.docker.com/_/golang>
+FROM golang:1.15.5-alpine as builder
+
+# app version and build date must be passed during image building (version without any prefix).
+# e.g.: `docker build --build-arg "APP_VERSION=1.2.3" --build-arg "BUILD_TIME=$(date +%FT%T%z)" .`
+ARG APP_VERSION="undefined"
+ARG BUILD_TIME="undefined"
+
+# arguments to pass on each go tool link invocation
+ENV LDFLAGS="-s \
+-X github.com/spiral/roadrunner/cmd/rr/cmd.Version=$APP_VERSION \
+-X github.com/spiral/roadrunner/cmd/rr/cmd.BuildTime=$BUILD_TIME"
 
 COPY . /src
 
 WORKDIR /src
 
+# download dependencies and compile binary file
 RUN set -x \
-    && apt-get update -y \
-    && apt-get install -y bash git \
-    && go version \
-    && bash ./build.sh \
-    && test -f ./.rr.yaml
+    && go mod download \
+    && go mod verify \
+    && CGO_ENABLED=0 go build -trimpath -ldflags "$LDFLAGS" -o ./rr ./cmd/rr/main.go
```

Comment:
To use the advantage of docker caching layers, it's better to separate this command into 2:
RUN go mod download + go mod verify
RUN build

Reply:
done